tools/c7n_salactus/supervisord.conf

```diff
@@ -15,7 +15,7 @@ loglevel=info                ; (log level;default info; others: debug,warn,trace
 user=ubuntu
 pidfile=/tmp/supervisord.pid ; (supervisord pidfile;default supervisord.pid)
 nodaemon=false               ; (start in foreground if true;default false)
-;minfds=8192                  ; (min. avail startup file descriptors;default 1024)
+minfds=4096                  ; (min. avail startup file descriptors;default 1024)
```

Comment:
keep in mind this is per process, you may run into max for the system unless you separately (system config) adjust that limit.